CMakeLists.txt

```diff
@@ -11,6 +14,13 @@ find_package(PkgConfig)
 # Define GNU standard installation directories
 include(GNUInstallDirs)
 
+# Generate code coverage counters and HTML report
+if ( CMAKE_BUILD_TYPE STREQUAL "Coverage" OR CMAKE_BUILD_TYPE STREQUAL "Debug" )
```

Comment:
Sometimes you leave a space between the `( CMAKE` and sometimes you don't `(CMAKE_CXX_FLAGS)`. Could you please change the commit so these are consistent?